lib/facility_factory.rb

```diff
@@ -0,0 +1,37 @@
+class FacilityFactory
+
+  def initialize
+  end
+
+  def create_facilities(registration_location)
+    registration_location.map do |registration|
+      Facility.new(registration_location)
```

Comment:
So if you put a pry on line 6 1/2, you will see that `registration_location` is a hash but it is the hash that is passed if from 
```ruby    
@facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014'})
```
We are wanting to create entirely new `Facility` objects from the data that we are getting from our external api
```ruby
@or_dmv_office_locations = DmvDataService.new.or_dmv_office_locations
```
So this one method should be able to take in any facilities data and format it the way that we want it. 
So instead of creating methods for each different state (what if we decided we needed all 50 states; that wouldn't be very DRY), you would want to have one method that was dynamic. 
So you may need to create helper methods that format things like name, address, and phone number depending on the state info you are pulling from that you could call in your `create_facilities` method.